api/rosetta/construction.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2021 Optakt Labs OÜ
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+package rosetta
+
+const (
+	// FlowSignatureAlgorithm denotes the signing algorithm supported by Flow.
+	FlowSignatureAlgorithm = "ecdsa"
+)
+
+// TODO: We could consider moving Data and Construction APIs to separate packages.
+// TODO: when the construction API crystalizes, make all the error returns consistent
+// TODO: as a WIP thing - treat the sender as the proposer and the payer. Consider allowing more granular control.
+// TODO: set gas limit for the transaction.
+// TODO: get account sequence number for the metadata endpoint
```

Comment:
Account sequence number fetching is TBD. I believe this is where our Access API would come into play since we need to get the specific accounts data.

Reply:
Yes, that's what I mean with "our" Access API. The only place where the actual, Flow Access API comes into play is when submitting the transaction to the network.

For all other stuff, we rely on the DPS API.